images/spz/spz/templates/internal/registrations.html

```diff
@@ -22,4 +22,22 @@ <h3 class="ui dividing header">Import</h3>
         <button type="submit" class="fluid ui positive button">Einlesen</button>
     </form>
 </div>
+
+<div class="row">
+    <h3 class="ui dividing header">Matrikelnummer/Kürzel-Überprüfung</h3>
+    <form class="ui form" method="post" action="{{ url_for('registrations_verify') }}" enctype="multipart/form-data">
+        {{ csrf_field() }}
+        {{ render_input(form.tag) }}
+        {% if tag_exists %}
+        <div class="ui positive message">
+            Matrikelnummer/Kürzel "{{ tag }}" ist in bereits in der Datenbank.
+        </div>
+        {% elif tag.data %}
```

Comment:
It's very unlikely that this is going to work. If [you don't pass a tag object at all](https://github.com/spz-signup/spz-signup/pull/62/files#diff-c33e317b7d37b8e87cb84f6fdc929f0aR195) then you try to access an attribute of `None` here. Notice you can access objects that are not defined (that results in `None`) but you cannot access attributes of objects that are not defined (= attributes of `None`).